setup.py

```diff
@@ -40,13 +40,13 @@
 INSTALL_REQUIRES = ['numpy >= 1.12.0',
                     'pandas >= 0.22.0',
                     'pytz',
-                    'requests']
+                    'requests',
+                    'scipy >= 1.2.0']
 TESTS_REQUIRE = ['nose', 'pytest', 'pytest-cov', 'pytest-mock',
                  'pytest-timeout', 'pytest-rerunfailures', 'pytest-remotedata']
 EXTRAS_REQUIRE = {
-    'optional': ['cython', 'ephem', 'netcdf4', 'nrel-pysam', 'numba',
-                 'pvfactors', 'scipy', 'siphon', 'statsmodels', 'tables',
-                 'cftime >= 1.1.1'],
+    'optional': ['ephem', 'cython', 'netcdf4', 'nrel-pysam', 'numba',
+                 'pvfactors', 'scipy', 'siphon', 'tables', 'cftime >= 1.1.1'],
```

Comment:
```suggestion
                 'pvfactors', 'siphon', 'statsmodels', 'tables', 'cftime >= 1.1.1'],
```

Reply:
thanks, I also forgot to remove `scipy` from the doc requirements.